main.js

```diff
@@ -0,0 +1,51 @@
+const recipe_object = {
```

Comment:
It would be much better and easier if your recipe object had the same form as the object in the exercise description like this:
```suggestion
const recipe_object = {
  id: 1,
  title: "Gløgg",
  picture_url:
    "https://upload.wikimedia.org/wikipedia/commons/thumb/e/e2/Gl%C3%B6gg_kastrull.JPG/800px-Gl%C3%B6gg_kastrull.JPG",
  ingredients: [
    { NAME: "Orange zest", AMOUNT: "0.5" },
    { NAME: "Water", AMOUNT: "200 ml" },
    { NAME: "Sugar", AMOUNT: "275 g" },
    { NAME: "Whole cloves", AMOUNT: "5" },
    { NAME: "Cinnamon sticks", AMOUNT: "2" },
    { NAME: "Spice", AMOUNT: undefined },
    { NAME: "Bottle of red wine", AMOUNT: "1" },
    { NAME: "Raisins", AMOUNT: "100 g" },
    { NAME: "Slipped Almonds", AMOUNT: "50 g" },
  ],
  description: "Mix everything, heat it, and you are good to go!",
};
```